avm/res/web/site/config--appsettings/main.bicep

```diff
@@ -35,7 +35,7 @@ param appSettingsKeyValuePairs object?
 
 var azureWebJobsValues = !empty(storageAccountResourceId) && !(storageAccountUseIdentityAuthentication)
   ? {
-      AzureWebJobsStorage: 'DefaultEndpointsProtocol=https;AccountName=${storageAccount.name};AccountKey=${storageAccount.listKeys().keys[0].value};'
+      AzureWebJobsStorage: 'DefaultEndpointsProtocol=https;AccountName=${storageAccount.name};AccountKey=${storageAccount.listKeys().keys[0].value};EndpointSuffix=core.windows.net'
```

Comment:
```suggestion
      AzureWebJobsStorage: 'DefaultEndpointsProtocol=https;AccountName=${storageAccount.name};AccountKey=${storageAccount.listKeys().keys[0].value};EndpointSuffix=${environment().suffixes.storage}'
```

Reply:
Nope it did work OK. But that function is nicer! ;)

Reply:
Interesting, Bicep linter should've raised its hand. Ah well...
For some context: The Linter warning would suggest to use the `environment()` function bcause it's cloud-agnostic (that is commercial vs gov)